plugins/kubernetes-common/api-report.md

```diff
@@ -3,9 +3,19 @@
 > Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
 
 ```ts
+import { ApiRef } from '@backstage/core-plugin-api';
 import { BasicPermission } from '@backstage/plugin-permission-common';
+import { CustomObjectsByEntityRequest as CustomObjectsByEntityRequest_2 } from '@backstage/plugin-kubernetes-common';
+import { DiscoveryApi } from '@backstage/core-plugin-api';
 import { Entity } from '@backstage/catalog-model';
+import { Event as Event_2 } from 'kubernetes-models/v1';
+import { FetchResponse as FetchResponse_2 } from '@backstage/plugin-kubernetes-common';
+import { IdentityApi } from '@backstage/core-plugin-api';
```

Comment:
`core-plugin-api` is frontend-only, so can't depend on it in a `-common` package. You'll likely want this kind of workaround, until we've figured out a place to share common types for these too: https://github.com/backstage/backstage/blob/53b7c7668d0f783e50359e091d63ddc429aee05b/packages/catalog-client/api-report.md#L87-L92

Reply:
have moved this